### ISSUE TITLE
"This web site is not currently available"

### ISSUE THREAD

**Issue by blueroses404** [floblik/YouScrobbler#20]
Hi. I love YouScrobbler. I've been using it for a really long time, and I have
a fairly large database of songs. I love being able to watch videos on it
without having to worry about tracking music along with it.

However, a few days ago, it suddenly stopped working, and I don't understand why.
I got this error:

![youscrobbler error2](https://user-images.githubusercontent.com/43664889/46190964-4cebe100-c2bb-11e8-8738-7d1c3ca25d22.PNG)

Out of the blue, it doesn't seem to work anymore. I tried uninstalling the userscript and re-uploading
my backup files, uninstalling and reinstalling Tampermonkey, authenticating my Last.fm user account
again. Everything. But nothing's working.

I messed with a lot of the settings just now, and it seems to have made it worse:

![youscrobbler error1](https://user-images.githubusercontent.com/43664889/46191046-94726d00-c2bb-11e8-8961-0fa2bce0bdea.PNG)

When I click on the [YouScrobbler site,](http://www.lukash.de/youscrobbler) it just says, "404 Not Found. The requested URL /youscrobbler was not found on this server." However, I seem to remember the linked site being like that even before, and I don't think that's new? (The userscript is really old at this point and has not been updated in a while)

Is the site/userscript/plugin just broken?

I'm really disappointed. Some help would be really appreciated, thanks.

**Comment by wilkowy** [comment]
Yeah, the whole website is wiped out (with http://youscrobbler2.lukash.de/scrobblesong/ ) so no more scrobbling. At least for now…

**Comment by floblik** [comment]
Sorry for the caused confusion and the late update. 

I had to take the server endpoint offline as someone was messing with it and I also was not able to keep updating YouScrobbler itself.
